packages/gatsby-image/gatsby-browser.js

```diff
@@ -0,0 +1,17 @@
+"use strict"
+
+exports.onClientEntry = () => {
+  // IntersectionObserver polyfill for gatsby-image (Safari, IE)
+  if (typeof window.IntersectionObserver === `undefined`) {
+    import(`intersection-observer`)
+  }
+
+  // Object-fit/Object-position polyfill for gatsby-image (IE)
+  const testImg = document.createElement(`img`)
+  if (
+    typeof testImg.style.objectFit === `undefined` ||
+    typeof testImg.style.objectPosition === `undefined`
+  ) {
+    import(`object-fit-images`)()
```

Comment:
I don't think this is correct? Shouldn't it be something like:
```js
import(`object-fit-images`).then(ObjectFitImages => ObjectFitImages())
```
?

Reply:
On my projects, it works as written, but I'm happy to implement the rewrite if it's more solid.

Reply:
> But with IntersectionObserver there might be timing issue - https://www.npmjs.com/package/intersection-observer: Be sure to include the polyfill prior to referencing it anywhere in your JavaScript code we might try to reference it earlier.

It's been working for me to load the IO polyfill in `gatsby-browser, but if you know of an earlier place we can load it, that would be great!